.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
-
+cli/cli
```

Comment:
It's conventional to use a `cmd` folder for your CLI; if you want, you could create a further subfolder with the name you want the binary to have. For example, if it should be called `go-history`, then put your `main.go` in `cmd/go-history`, and running `go build` will automatically create a binary with that name.

Alternatively, you can put it in just `cmd`, and then run `go build -o go-history`. Whichever you prefer!